source/frontend/src/features/mapSideBar/tabs/SectionStyles.tsx

```diff
@@ -68,6 +68,7 @@ export const ArrowDropUpIcon = styled(MdArrowDropUp)`
 
 export const StyledSummarySection = styled.div`
   background-color: ${props => props.theme.css.filterBackgroundColor};
+  padding-top: 1rem;
```

Comment:
This affects ~20 pages in the application, is this something that should be on all of those forms that consume this?

Reply:
I believe so, I did a naive test of the pages and it looked correct to me.